### ISSUE TITLE
[Feature Request] Enable offline Speech-to-Text and Text-to-Speech

### ISSUE THREAD

**Issue by JRMeyer** [jovotech/jovo-framework#1237]
👋 hi there!

## I'm submitting a...

- [ ] Bug report
- [x] Feature request
- [ ] Documentation issue or request
- [ ] Other... Please describe:

## Expected Behavior

Would be great to be able to test and debug a voice bot _without_ an internet connection. Offline STT and TTS (from @coqui-ai) would make this possible using the existing UX from the new `jovo debugger`

## Current Behavior

Currently there's no offline STT or TTS

**Comment by jankoenig** [comment]
Hi there. Thank you.

This is not on our immediate roadmap, but would be a great community contribution.

Coqui STT could be implemented as [Jovo ASR integration](https://www.jovo.tech/docs/asr).

**Comment by JRMeyer** [comment]
Hi @jankoenig -- just looked into the integration with Lex, and it would be considerably different with Coqui because the user would have their own server running. For example, the user might be running a simple server on their local desktop or they might have spun up a server on their AWS cloud, and using endpoints there. In either case, the API syntax and integration would be identical, but there would be an expectation that the user spins up the server themselves. Not too difficult, but I'm not sure if that's something the Jovo crowd would be interested in.

I think the biggest value add for Jovo users would be to be able to test out their voicebots locally, without having an ASR backend running on one of the providers (like Lex).

Thoughts?

**Comment by jankoenig** [comment]
This could work similar to our [Snips NLU integration](https://www.jovo.tech/marketplace/nlu-snips) where people also have to run their own servers.

An integration like this would also be useful for our web starters:
- https://github.com/jovotech/jovo-starter-web-standalone
- https://github.com/jovotech/jovo-starter-web-standalone-react

**Comment by JRMeyer** [comment]
Yeah, I think a general setup mirroring the Snips approach would work nicely. You know of anyone in your community who might like to hack on this? We're happy to offer support/guidance for using the Coqui tools.

**Comment by rubenaeg** [comment]
I think I could give this a spin :)

**Comment by rmtuckerphx** [comment]
@JRMeyer Are there any developer docs on the Coqui APIs for STT and TTS using Node.js or REST?
